scripts/scenario_03_edgeofspace.lua

```diff
@@ -238,9 +257,42 @@ function update(delta)
 			Central_Command.mission_state = 2
 		end
 	end
+
+	--When the Apollo is docked with EOS_Station a message is received from the Technical Officer advising that his team is beginning their inspection

```

Comment:
I'm not aware of any way to send messages to Relay without an actual entity existing with this alias, so I just hid a Tug and Station way off in the distance.